packages/skip/src/swap.rs

```diff
@@ -55,10 +55,20 @@ pub struct LidoSatelliteInstantiateMsg {
 #[cw_serde]
 pub enum ExecuteMsg {
     Receive(Cw20ReceiveMsg),
-    Swap { routes: Vec<Route> },
-    TransferFundsBack { swapper: Addr, return_denom: String },
-    AstroportPoolSwap { operation: SwapOperation }, // Only used for the astroport swap adapter contract
-    WhiteWhalePoolSwap { operation: SwapOperation }, // Only used for the white whale swap adapter contract
+    Swap {
+        routes: Vec<Route>,
+    },
+    TransferFundsBack {
+        swapper: Addr,
+        return_denom: String,
+    },
+    AstroportPoolSwap {
+        operation: SwapOperation,
+        offer_asset: Option<Asset>,
+    }, // Only used for the astroport swap adapter contract
```

Comment:
let's move these comments above the variant now, inline comments look good in one liners, not so much w/ multiliner